### ISSUE TITLE
Table 'mydb.sessions' doesn't exist

### ISSUE THREAD

**Issue by vingorius** [pallets-eco/flask-session#121]
When using sqlalchemy with mysql, http://localhost:port/get shows below error.
I think there's no table named 'sessions' in mysql.
how can i create it or where's create table statement.
thank you in advance.

> sqlalchemy.exc.ProgrammingError
> 
> sqlalchemy.exc.ProgrammingError: (mysql.connector.errors.ProgrammingError) 1146 (42S02): Table 'gentok.sessions' doesn't exist
> [SQL: SELECT sessions.id AS sessions_id, sessions.session_id AS sessions_session_id, sessions.data AS sessions_data, sessions.expiry AS sessions_expiry 
> FROM sessions 
> WHERE sessions.session_id = %(session_id_1)s 
>  LIMIT %(param_1)s]
> [parameters: {'session_id_1': 'session:0497ac5b-323d-4496-95a0-c26414f49035', 'param_1': 1}]
> (Background on this error at: http://sqlalche.me/e/f405)
>

**Comment by henrylamchan** [comment]
Was just looking into this myself. Should there or is there any documentation for this? I'm assuming we'll need a table similar to what's here: https://github.com/fengsp/flask-session/blob/master/flask_session/sessions.py#L477

**Comment by AllanChain** [comment]
If you don't set `SESSION_SQLALCHEMY`, flask_session will create it's own `db` instance, and the `session` table is created with this `db` instance. Thus `session` table will not be created by calling `created_all` with your `db` instance.

Quick fix:

```python
db.init_app(app)
app.config['SESSION_SQLALCHEMY'] = db
Session(app)
```

**Comment by erm3nda** [comment]
Im not sure about those assumptions.

Case of my, I didn't set `app.config['SESSION_SQLALCHEMY'] = db`, session did not created it's own instance, session complained about sessions table not found (in my current db).

I've just added one more `create_all()` call for it.

```
# function to create database in first attemtp to use it
@app.before_first_request
def startup():
    # create tables from our models
    db.create_all()
    # create session tables
    app.session_interface.db.create_all()
```

This will create session tables in an existent database, no need to delete it.
Maybe that auto call to create_all() has been removed at some point.

Btw, what I am facing is a lot of problems with python3 and this library.

**Comment by Lxstr** [comment]
This should be fixed now from 0.6.0. However, it might be good to have a warning run when no SESSION_SQLALCHEMY etc is set.
